src/Catel.Core/Catel.Core.Shared/IoC/Interfaces/IServiceLocator.cs

```diff
@@ -115,6 +115,15 @@ public interface IServiceLocator : IServiceProvider, IDisposable
         /// <returns><c>true</c> if the <paramref name="serviceType" /> type is registered as singleton, otherwise <c>false</c>.</returns>
         bool IsTypeRegisteredAsSingleton(Type serviceType, object tag = null);
 
+        /// <summary>
+        /// Determines whether the specified service type is registered tag regardless.
+        /// </summary>
+        /// <param name="serviceType">The type of the service.</param>
+        /// <returns><c>true</c> if the specified service type is registered; otherwise, <c>false</c>.</returns>
+        /// <remarks>Note that the actual implementation lays in the hands of the IoC technique being used.</remarks>
+        /// <exception cref="ArgumentNullException">The <paramref name="serviceType"/> is <c>null</c>.</exception>
+        bool IsTypeRegisteredTagRegardless(Type serviceType);
```

Comment:
In what context is "regardless" meant? I'm not sure whether it's the right name for this method.